pkg/tfvars/openstack/bootstrap_ignition.go

```diff
@@ -52,6 +56,7 @@ func createBootstrapSwiftObject(cloud string, bootstrapIgn string, clusterID str
 	if err != nil {
 		return "", err
 	}
+	logrus.Debugf("Object %s was created.", objID)
```

Comment:
This objID is supposed to be secret :) We do not recommend to publish it.

Reply:
\o/ didnt know that